web-app/src/static/js/video_analysis.js

```diff
@@ -45,41 +40,79 @@ const taskForm = (formName, doPoll, report) => {
 }
 
 taskForm("video-upload-form", true, data => {
-  const progressbar = document.getElementById("progressbar")
+  const progressbar = document.getElementById("progressbar");
+  const tacticalboard = document.getElementById("tacticalboard");
+  const fileInput = document.querySelector('input[type="file"]');
+  const videoPlayer = document.getElementById('videoplayer');
+
+
   if (data === null) {
     console.log("uploading...")
+    progressbar.value = 0;
```

Comment:
Note: I've seen issues from line 53 (`progressbar.value = data["value"]["status"]`) and used a more defensive code in my branch:
```
progressbar.value = data["value"]?.["status"] ?? 0
```
This probably happens because there's a race condition between web app starting a celery task, and the UI starting to listen for task status - which may happen before the task even starts.